pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -192,8 +192,9 @@ class Build {
                             parameters: params
                     
                     // Output notification of downstream failure (the build will fail automatically)
-		    if (signJob.getResult() == 'FAILURE') {
-                        context.println "ERROR: downstream sign_build FAILED. See ${signJob.getAbsoluteUrl()}"
+					def jobResult = signJob.getResult()
```

Comment:
whitespace seems out for all of the changes.

Reply:
Note to self, never use the github editor ever again. It was fine when I typed it 😭